sympy/stats/rv.py

```diff
@@ -223,7 +223,10 @@ class RandomSymbol(Expr):
     convenience functions Normal, Exponential, Coin, Die, FiniteRV, etc....
     """
 
-    def __new__(cls, pspace, symbol):
+    def __new__(cls, pspace, symbol=None):
```

Comment:
We should discuss this in a separate issue. I disagree that it's worth breaking the API like this to keep the code clean.